framework/src/executioners/FEProblemSolve.C

```diff
@@ -190,26 +208,9 @@ FEProblemSolve::FEProblemSolve(Executioner & ex)
   es.parameters.set<unsigned int>("linear solver maximum iterations") =
```

Comment:
add a few lines above:

// Set linear solve parameter in the equation system
// Nonlinear solve parameters are added in the DefaultNonlinearConvergence